src/utils/httpRequestParser.ts

```diff
@@ -179,7 +180,10 @@ export class HttpRequestParser implements RequestParser {
                         const inputFilePath = groups[1];
                         const fileAbsolutePath = await resolveRequestBodyPath(inputFilePath);
                         if (fileAbsolutePath) {
-                            combinedStream.append(fs.createReadStream(fileAbsolutePath));
+                            const buffer = await fs.readFile(fileAbsolutePath);
+                            const fileContent = buffer.toString('utf8');
```

Comment:
Here you simply convert the buffer to UTF-8 string, if the buffer encoding is not `utf8`, this would make the final request body incorrect

Reply:
what's the best way of doing it?

Reply:
I found that VSCode interally use the library `jschardet` to guess textual file content encoding (https://github.com/microsoft/vscode/blob/ce1462776f7ada735ba82ffec28ea774e617a927/src/vs/workbench/services/textfile/common/encoding.ts#L234). Together with the following advice, if we add the request level setting for this feature, maybe we can also support providing an optional argument `encoding`, if omit, we will guess the encoding for him/her. By default the setting is turned off.

Reply:
think we can default to utf8 and provide a way to override the default to keep it simple

Reply:
added those changes, can you check?